phpBB/composer.json

```diff
@@ -39,7 +39,7 @@
 		"composer/package-versions-deprecated": "^1.11",
 		"doctrine/dbal": "~3.3.6",
 		"google/recaptcha": "~1.1",
-		"guzzlehttp/guzzle": "~6.3",
+		"guzzlehttp/guzzle": " ^7.0",
```

Comment:
I take it guzzle had to be updated?

Reply:
Yeah, it's been throwing these depreciations too in background.